src/app/toollbar/toollbar.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-inverse">
+        <div class="container-fluid">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#" style="color:rgb(236, 160, 60); font-size: 40px">Number</a>
+            <a class="navbar-brand" href="#" style="color:rgb(230, 189, 101); font-size: 25px; padding: 9px">Ninja</a>
+          </div>
+          <ul class="nav navbar-nav">
+            <li><a href="#" style="color:yellow">HOME</a></li>
+            <li><a href="#" style="color:yellow">ASSIGNMENTS</a></li>
+          </ul>
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#"><span class="glyphicon glyphicon-user" style="color:cyan"></span> Log Out</a></li>
+          <!---- <li><a href="#"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>-->
```

Comment:
Remove line 13 comment and move styling tags to css file.

Reply:
Cannot move styling. Refer to previous comment